tests/guardrails/max_bolus.tests.js

```diff
@@ -0,0 +1,62 @@
+const { limits } = require('../../src/index');
+
+var maxBolusTests = (test) => {
+    var screen;
+    it('open', async () => {
+        screen = await test.settingsScreen.OpenDeliveryLimitsScreen();
+    });
+    it('cannot set above the max limit', async () => {
+        await screen.Apply({
+            maxBolus: limits.bolusDelivery.max.limit + limits.bolusDelivery.step,
+            maxBasalRate: '1.0'
```

Comment:
[nit] Internally to the `Apply` function can we wrap the `maxBasalRate` value with `String(value)` so this could be an actual number and not a string?

Reply:
yeah - detox wants strings but they are numeric values. I will update to keep consistent